File: Iniciante/PHP/exercise_1043.php
<?php

function isTriangle($valorA, $valorB, $valorC) {
    return ($valorA+$valorB > $valorC) && ($valorB+$valorC > $valorA) && ($valorC+$valorA > $valorB);
}

$valores = readline();

list($valorA, $valorB, $valorC) = sscanf($valores, "%f %f %f");

if (isTriangle($valorA, $valorB, $valorC)) {
    $perimetro = $valorA+$valorB+$valorC;

    print_r("Perimetro = ".number_format($perimetro, 1)."\n");
} else {
    $areaTrapezio = ($valorA+$valorB)*$valorC/2;

    print_r("Area = ".number_format($areaTrapezio, 1)."\n");
}

//Concluído!

File: Iniciante/Python/exercises from 1000 to 1099/exercise_1043.py
def isTriangle(valorA, valorB, valorC):
    isAPlusBBiggerThanC = valorA+valorB > valorC
    isBPlusCBiggerThanA = valorB+valorC > valorA
    isAPlusCBiggerThanB = valorA+valorC > valorB
    return isAPlusCBiggerThanB and isBPlusCBiggerThanA and isAPlusBBiggerThanC

valorA, valorB, valorC = [float(x) for x in input().split()]

if isTriangle(valorA, valorB, valorC):
    perimetro = valorA+valorB+valorC
    print('Perimetro = '+ "{:.1f}".format(perimetro))
else:
    area = ((valorA+valorB)*valorC)/2
    print('Area = '+ "{:.1f}".format(area))
